content/lua/concepts/operators/operators.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: 'Operators' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Operators are special characters that can carry out mathematical tasks and resolve logical expressions.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Operators'
+  - 'Math'
+  - 'Numbers'
+  - 'Logical'
+  - 'Arithmetic'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+**Operators** are special characters that can carry out mathematical tasks and resolve logical expressions.
+
+## Arithmetic Operators
+
+**Arithmetic Operators** can be used to carry out mathematical operations:
+
+```lua
+-- Addition. Adds two numbers.
+print(1 + 1) -- "2"
+
+-- Subtraction. Subtracts two numbers
+print(2 - 1) -- "1"
+
+-- Multiplication. Multiplies two numbers.
+print(2 * 2) -- "4"
+
+-- Division. Divides two numbers.
+print(4 / 2) -- "2"
+
+-- Exponentiation. Performs an exponential operation
+print(3 ^ 3) -- "27"
+
+-- Modulus. This is a special operator that returns the remainder of a division expression.
+print(16 % 3) -- "1" (16 divided by 3 would leave a remainder of 1)
+print(25 % 5) -- "0" (25 divides evenly into 5)
+
+-- Unary negation. This is not the same as subtraction, as it can be placed in front of a number/expression to negate it.
+print(-1) -- "-1"
+print(5 + (-1)) -- "4"
+print(-(4 * 4)) -- "-16"
+```
+
+
+
+## Compound Assignments
+
+**Compound Assignments** can be used to carry out mathematical operations and assign them to **variables**. To prevent errors, the initial value assigned to the variable should align with the input we are updating our variable with.
```

Comment:
```suggestion
Compound assignments can be used to carry out mathematical operations and assign them to variables. To prevent errors, the initial value assigned to the variable should align with the input used to update the variable.
```

Reply:
we omit the use of first and second person pronouns (I, you, we) in docs